src/organisms/image-upload/index.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from "react";
+import axios from "axios";
+import { Input } from "@mui/material";
+import PhotoCamera from "@mui/icons-material/PhotoCamera";
+import Box from "@mui/material/Box";
+import IconButton from "@mui/material/IconButton";
+import styled from "@emotion/styled";
+
+const CLOUDNAME = process.env.NEXT_PUBLIC_CLOUDNAME;
+const PRESET = process.env.NEXT_PUBLIC_PRESET;
+
+const ImageUpload = () => {
+	const [image, setImage] = useState(null);
+
+	const Image = styled.img`
+		max-width: 40%;
+		max-height: 40%;
+		border: 4px solid #e4eaeb;
+	`;
```

Comment:
this should lay outside of your ImageUpload component, also you could think of using the next.js Image Component, as it brings a lot of advantages.